cmd/awscollector/Dockerfile

```diff
@@ -4,63 +4,45 @@
 ################################
 FROM alpine:latest as build 
 
-ENV GOPROXY=direct
-
 # update cert
 RUN apk --update add ca-certificates
 
-# install libs
-RUN apk add --update bash git make build-base
-
-# config go
-COPY --from=golang:1.17-alpine /usr/local/go/ /usr/local/go/
-ENV PATH="/usr/local/go/bin:${PATH}"
-
 WORKDIR /workspace
 
-# copy artifacts
-COPY cmd  /workspace/cmd
-COPY config.yaml /workspace/config.yaml
-COPY pkg /workspace/pkg
-COPY tools /workspace/tools
-COPY VERSION /workspace/VERSION
-COPY .git /workspace/.git
-COPY Makefile /workspace/Makefile
-COPY config /workspace/config
+ARG TARGETPLATFORM
+RUN echo "Targate platform is $TARGETPLATFORM"
+ARG VERSION=unknown
+RUN echo "Version is $VERSION"
 
-#Copy binary if exist else build binary
-COPY go.* build/linux/aoc_linux_x86_6* /workspace/
-RUN if [[ -f /workspace/aoc_linux_x86_64 ]] ; \
-    then \
-        echo Copying binary \
-        && mkdir -p /workspace/build/linux \
-        && cp /workspace/aoc_linux_x86_64 /workspace/build/linux/aoc_linux_x86_64 ; \
-    else \
-        echo Building binary \
-        && go mod download  \
-        && make amd64-build ; \
-    fi
+# copy artifacts
+# always assume binary is created
+COPY build/linux/* /workspace/
+COPY cmd/awscollector/copy_docker_binary.sh  /workspace/cmd/awscollector/copy_docker_binary.sh
+RUN chmod +x cmd/awscollector/copy_docker_binary.sh
+RUN cmd/awscollector/copy_docker_binary.sh
 
 ################################
 #	Final Stage            #	
 #			       #	
 ################################
 FROM scratch
 
+ARG VERSION=unknown
+LABEL version=$VERSION
```

Comment:
This should use the [OCI standard labels](https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys).  While we're at it we should see if there are any others that can easily be populated.

Reply:
We can use docker-metadata action to populate some standard labels automatically

https://github.com/docker/metadata-action

Reply:
From our chat with @anuraaga I will use docker metadata